pkg/netcat/util.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package netcat
+
+import (
+	"fmt"
+	"io"
+	"log"
+)
+
+// Logf logs a message if the verbose flag is set.
+func Logf(nc *NetcatConfig, format string, args ...interface{}) {
```

Comment:
This is not really how you want to do this IMHO if you're going a package. Maybe see pkg/boot/boot.go

Reply:
I've rewritten the config to use ulog and its features.